quality_control/views/product_template_view.xml

```diff
@@ -11,8 +11,9 @@
         <field name="name">product.template.qc</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="groups_id" eval="[(4, ref('quality_control.group_quality_control_manager'))]" />
```

Comment:
A user should view also the triggers IMO

Reply:
DONE